README.md

```diff
@@ -62,7 +62,7 @@ Even if you find a single-character typo, we're happy to take the change! Althou
 
 ## Per-build NuGet packages
 
-Per-build verions of our NuGet packages are available via this URL: `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json `
+Per-build [versions](https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-tools&view=versions&package=FSharp.Compiler.Service&protocolType=NuGet) of our NuGet packages are available via this URL: `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json `
```

Comment:
Pre* build?

Reply:
Though maybe this should be more like 'prerelease' versions?

Reply:
I think "per-build" makes sense here, as in "one package per build (of main)"?

Reply:
I was under the impression there are prereleases from a few different branches (possibly release/FCS as well)? This is based on version number change propagation as well as pulling FCS 40 prereleases in FSAC for a bit and seeing wildly different API surface areas day to day 😂